ols/src/llms/llm_loader.py

```diff
@@ -139,16 +156,12 @@ def _openai_llm_instance(self) -> LLM:
             )
             raise e
         params: dict = {
-            "base_url": (
-                self.provider_config.url
-                if self.provider_config.url is not None
-                else "https://api.openai.com/v1"
-            ),
-            "openai_api_key": self.provider_config.credentials,
+            "base_url": self._get_llm_url("https://api.openai.com/v1"),
+            "openai_api_key": self._get_llm_credentials(),
```

Comment:
it seems a bit odd and non-discoverable to me that we wouldn't just pass this value in as an arg to `_openai_llm_instance`.  Same for the url.